src/dlsproto/client/legacy/internal/helper/GroupRequest.d

```diff
@@ -88,3 +93,40 @@ public template GroupRequest ( Request )
         GroupRequest;
 }
 
+/*******************************************************************************
+
+    Instantiates a GroupRequest instance wrapped around the provided Request
+    instance. Used to allow template parameter deduction.
+
+    Params:
+        Request = type of the request
+        req = request to manage using GroupRequest
+
+    Returns:
+        instance of the GroupRequest!(Request) managing the req instance.
+
+*******************************************************************************/
+
+public GroupRequest!(Request) makeGroupRequest(Request)(Request req)
+{
+    return new GroupRequest!(Request)(req);
+}
+
+unittest
```

Comment:
`///`?

Reply:
I didn't put this one for the documentation purposes, though.

Reply:
Sure. It looks like a usage example as well, though. Just a minor suggestion. I'll merge anyway, if you don't think it's worth it ;)

Reply:
No, I think it's worth it - I have this dreadful epoll and client instantiation, and there's usage example in the module docs.

Reply:
You mean "I think it's not worth it" (i.e. there's no point adding `///`).

Reply:
Yes, sorry, typo!